src/App.js

```diff
@@ -13,47 +16,69 @@ export default class App extends React.Component {
     this.state={
       searchQuery: '',
       location: {},
+      error: false
     }
   }
 
-  getlocation = async (event) => {
+  getLocationData = async (event) => {
     event.preventDefault();
-    console.log(this.state.searchQuery)
-    console.log(process.env.REACT_APP_CITY_KEY)
-    var coord = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.searchQuery}&format=json`;
+    try{
+    const coord = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.searchQuery}&format=json`;
     const response = await axios.get(coord);
-    console.log(coord);
     const location = response.data[0];
-    console.log(location);
 
     this.setState({ 
       location,
+      error: false,
+    });
+    } catch(error) {
+      this.setstate({error: true})
+    };
+    var coord2 = `https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_CITY_KEY}&center=${this.state.location.lat},${this.state.location.lon}&zoom=12&format=jpeg`;
+    const response2 = await axios.get(coord2);
+    const mapLocation = response2.config.url;
+    console.log(mapLocation);
+
+    this.setState({ 
+      mapLocation,
     })
   
   }; 
-
+  
   render(){
     return (
       <>
-      <Form>
-        <Form.Label>
+      <Form
+        >
+        <h1>
           Enter a City Name
-        </Form.Label>
+        </h1>
         <Form.Control onChange={ event => this.setState({ searchQuery: event.target.value})}
-        type="text" placeholder="type in a city"/>
-        <Button onClick={this.getlocation} variant="primary" type="Submit">Explore?</Button>
-
-        {this.state.location.place_id &&
-          <Form.Text>City Name: {this.state.location.display_name}</Form.Text>
-        }
-        {this.state.location.place_id &&
-          <Form.Text>Latitude: {this.state.location.lat}</Form.Text>
+        type="text" placeholder="Type in a City"/>
+        <Button onClick={this.getLocationData} variant="primary" type="Submit">Explore?</Button>
+        </Form>
+      <Col>
+      <Card style={{ margin: '20px', width: '20 rem' }}>
+        <Card.Img variant="top" src={this.state.mapLocation} />
+        <Card.Body>
+          {this.state.location.place_id &&
+          <Card.Title>City Name: {this.state.location.display_name}</Card.Title>
+          }
+          {this.state.location.place_id &&
+          <Card.Text>Latitude: {this.state.location.lat}</Card.Text>
+          }
+          {this.state.location.place_id &&
+          <Card.Text>Longitude: {this.state.location.lon}</Card.Text>
+          }
+        </Card.Body>
+      </Card>
+      </Col>
+        <Container>
+        {this.state.error &&
+          <h2>'Please put in a valid City Name'</h2>
         }
```

Comment:
Nice work with the cards! They look pretty good upon rendering. A nice way to go to display the information.